doc/content/tutorials/KRUSTY.md

```diff
@@ -0,0 +1,885 @@
+# KRUSTY
+
+In this tutorial, you will learn how to:
+
+- Couple OpenMC via temperature and heat source feedback to MOOSE for the KRUSTY reactor
+- Control the resolution of physics feedback received by OpenMC
+
+To access this tutorial,
+
+```
+cd cardinal/tutorials/krusty
+```
+
+## Geometry and Computational Models
+
+This model consists of a multiphysics simulation of the KRUSTY reactor, one of the simplest space power reactor concepts ever proposed. The relevant dimensions are summarized in [table1] and in [krusty_solid_mesh] shows the geometry and gap thicknesses. 
```

Comment:
You may want to add a few sentences which discuss the geometric simplifications you are making (e.g., what is being neglected compared to the real system?). I'd also suggest to add a sentence with something along the lines of "Ongoing work benchmarking Cardinal against KRUSTY will relax these simplifications."